docs/api/system-preferences.md

```diff
@@ -106,6 +106,15 @@ This is necessary for events such as `NSUserDefaultsDidChangeNotification`.
 
 Same as `unsubscribeNotification`, but removes the subscriber from `NSNotificationCenter`.
 
+### `systemPreferences.registerDefaults(defaults)` _macOS_
```

Comment:
Is this method any different from `setUserDefault` except the fact it accepts multiple key/value "pairs"?

Reply:
yes, but typically a method that does this (registers multiple key/value) pairs is named `registerDefaults` so I think we might want to name it this since it's what most developers will be familiar with